cmd/server/app/options/options.go

```diff
@@ -139,6 +148,7 @@ func (o *ProxyRunOptions) Print() {
 	klog.V(1).Infof("KubeconfigBurst set to %d.\n", o.KubeconfigBurst)
 	klog.V(1).Infof("ProxyStrategies set to %q.\n", o.ProxyStrategies)
 	klog.V(1).Infof("WarnOnChannelLimit set to %t.\n", o.WarnOnChannelLimit)
+	klog.V(1).Infof("CipherSuites set to %q.\n", o.CipherSuites)
```

Comment:
Might be nice to print what the default suite is as a follow up if that's not too difficult.